nscplugin/src/main/scala/scala/scalanative/nscplugin/NirDefinitions.scala

```diff
@@ -21,7 +21,7 @@ trait NirDefinitions { self: NirGlobalAddons =>
     lazy val PtrLoadMethod   = getDecl(PtrClass, TermName("unary_$bang"))
     lazy val PtrStoreMethod  = getDecl(PtrClass, TermName("unary_$bang_$eq"))
     lazy val PtrAddMethod    = getDecl(PtrClass, TermName("$plus"))
-    lazy val PtrSubMethod    = getDecl(PtrClass, TermName("$minus"))
+    lazy val PtrSubMethods   = getDecl(PtrClass, TermName("$minus")).alternatives
```

Comment:
Given overloaded symbol, it gives you possible concrete alternatives. In this case `-` is overloaded for two types of pointer arithmetics.